### ISSUE TITLE
readAloud-valence-jess

### ISSUE THREAD

**Issue by jessb0t** [NDCLab/brainBox#4]
See readme file in rwe-valence-jess folder.

**Comment by jessb0t** [comment]
For transparency, posting some initial (pre-brainBox) feedback from @georgebuzzell on a sample stimuli text; this feedback was integrated into the brainBox issue here:
>[Jess] think this is really great! We should discuss more about what we are looking for in this study though. If we want to study the "switch" then we will need a lot of switch points. I think that what would be good is to build up one emotion for only a few sentences, then switch for a few, then switch back after a few, etc. Basically, we want to go as short as possible preswitch, while still building up that "mood state", it will take piloting to figure out how much buildup we need. Likely, the first study we run will be basically testing that question. So, we will vary the amount of time between switches, and see how much time is optimal to get switch effects.
>
>we also need to think critically about what will be our primary measure of interest. Will we start with just behavior and look at lag to vocolization and/or accuracy pre vs post switch? or, are we trying to use eeg markers as well, or both?

**Comment by georgebuzzell** [comment]
@jessb0t I think this is great. In general, I think we should try to focus in on 1-2 of the hypotheses for the initial project. Relatedly, we can't confidently make predictions about any neural effects right now, because we have not yet shown any control-related effects at the neural level in the real world error task.

Meta-comment (i.e. comment about brain box itself): for the template, I think we should add a required section after "question" that is called "summary" which asks the user to describe in 150 words or less, in simple language, the background, design, and goals of the project are. Then, the next section in the template should be "lit review and background", then hypotheses and the rest of the established template.


> **Question:**
> How does semantic valence interact with cognitive control in a real-world context?

Can we focus the question a bit more? I can make suggestions, but also, what are the most interesting sub-question(s) for you?

> 
> **Hypotheses:**
> _Behavioral Effects of Valence_
> **Hypothesis 1:** Positivity advantage would impact reading speed and error rate in valenced passages with faster speeds and fewer errors in positive passages than negative passages.

in the lit review, you describe prior work in line with this, but for simple, single word presentations. is there no work to show something similar at the passage level? If no, then this is "low hanging fruit" and an obvious thing to test. But, please be sure it has not been done?

> 
> * Are there differences in reading speed between negatively and positively valenced passages? Does presentation order matter?
> * Are there differences in the error rate (anticipatory errors, perseverations, etc.) between negatively and positively valenced passages? Does presentation order matter?

these are good, but can you please be more specific about how readng speed and accurcy will be measures? i.e. provide details on how you end up with an actual number.

expand on what is meant by "does presentation order matter"? I.e. exactly how will this be tested?

> 
> **Hypothesis 2:** Semantic priming effects would benefit positive>negative switches more than negative>positive, with a greater likelihood of hesitation or articulatory error at a negative>positive switch than a positive>negative switch.
> 
> * Is the hesitation rate at switches different between positive>negative and negative>positive switches?
> * Are there increased articulatory errors around one type of switch?  Given that participants will have access to all the text and may "read ahead" to begin encoding, we may need to look at the words prior to the switch, the switch itself, and the words immediately following the switch.

Agree that you need to look at multiple words before/after swtich. This relates back to exactly how these measures are being quantified.

how many switches will there be?
> 
> _Conflict Monitoring and Instantiation of Control_
> **Hypothesis 3:** Semantic integration of valenced words, when switching suddenly between a positive/neutral word set to a negative word (or vice versa), would show similar neural activity to that deployed during instantiation of cognitive control at the identification of a conflict, with recruitment of proactive control after the recognition potential but before onset of articulation of the switch word, and this should be more pronounced when switching from an individual's current mood state to the reverse; when increased control is absent, we would expect an increased likelihood of articulatory error on or around the switch word.
> 
> * Do we see increased MFC-LFC connectivity after presentation of the valence switch but before onset of articulation? If so, is likelihood of articulatory error decreased around the switch word?
> * Do we see any difference in MFC-LFC connectivity at presentation of positive>negative versus negative>positive valence switches? Is there a relationship with the participant's self-assessed mood state?

I would suggest leaving neural questions out of the initial study, and only bringing those in once the basic behavioral paradigm and effects are working (which would be a very nice paper on their own).

> 
> _Inhibitory Control and Mood Incongruence_
> **Hypothesis 4:** Affect serves as a "filter" through which an individual experiences the world; therefore, incongruence between the individual's mood state and the valence of words to be read aloud would force some level of inhibitory control in order to ignore one's mood state while encoding and articulating valenced words.
> 
> * If valenced section of a passage is incongruent with the participant's current baseline mood, is greater proactive control deployed or, in the absence of increased control, a rise in articulatory mistakes observed?
> * Given the importance of affect on social interactions, do we replicate Buzzell et al.'s (2019) finding that only error monitoring is up-regulated in the social condition or do we also find up-regulation of conflict monitoring, either at a switch word (stimulus internal conflict) or when incongruence between word valence and participant's mood state is increased (delta between ANEW score of switch word and participant's self-assessed mood score)?

I really like the idea of looking at the effect of the reader's initial mood state on reading. I think that can be studied initially without bringing in neural measures, as you can look at how the behavioral effects change not only as a function of global valence of a given text, or, when looking at post/pre switch, but how those effects interact with the intial state (and trait) mood/anxiety measures.

> 
> _Exploratory_
> Are we able to see the recognition potential of each word in a reading aloud task? If so, is there any difference in amplitude between positive, negative, and neutral words? Do we see any difference in recognition potential amplitude when the participant recognizes a valence switch word? Does switch direction have any impact on amplitude?
> 
> **Lit Review:**
> (See literature/lit-review.xlsx.)
Link?

> 
> Emotion has modulatory effects on cognition and behavior, with processing advantages often seen for positive, visually-presented words:
> 
> * slower processing of negative stimuli (versus positive) as seen by slower response time on negative target words in lexical decision tasks (Kazanas and Altarriba (2015))

-any work to show how this differs based on individual differences? I think there might be work on how this differs for anxious/non-anxious. I would be interested in effects based on both trait and state anxiety. 

> * positive primes in lexical decision tasks ostensibly cause spreading activation in the semantic network, thereby shortening response time for related target words but slowing response for unrelated target words; negative primes seem to cause an emotional reaction inhibition that precludes any change in response time for related/unrelated targets but enables faster processing for unrelated targets than a positive prime, with its larger network activation (Sass et al. (2012))
> * smaller N400 amplitudes for pleasant v. unpleasant adjectives in silent reading may indicate that pleasant words are semantically-integrated more easily, possibly as a result of congruency with default, mildly positive mood states (Herbert et al. (2008))
> 
> EEG evidence has linked to inhibitory control and social motivation in the classic flanker test:
> 
> * greater MFC-LFC connectivity before a response is linked to reactive control and improved performance on current task whereas greater MFC-LFC connectivity after a response is linked to proactive control and improved performance for the following task (Buzzell et al. (2019))
> * social motivation increases error monitoring, not conflict monitoring (Buzzell et al. (2019))
> 
> Further research still required on:
> 
> * prior EEG work involving affect/valence,
-i actually think you might want to drop eeg from the initial study. it will be a lot just to get the behavior working. And, if you have nice behavioral effects, that will be a nice advance already (and can then lead to a follow-up that naturally incorporates eeg). If you really want to include eeg, then I think it needs to build directly on eeg effects we see with the initial eeg rwe study.
> * prior work involving the N400 ERP,
-you probably will not be able to look at erps in a "real world reading task". It is possible, but also less likely for something like the n400 which in my experience does not have a huge signal to noise ratio. We are not even sure yet how well we will be able to look at time-frequency just yet. 
> * the recognition potential.
-what is this?
> 
> **Design:**
> Using the read-aloud task from rwe-alpha, special texts are created that are designed to evoke certain mood states in the reader (positive/negative) and include a valence "switch point" where the mood of the text suddenly reverses. (See stimuli/pos-neg_dolphins.txt for an example.) Requests for [ANEW and ANET](https://csea.phhp.ufl.edu/media.html) have been submitted to assist in text design. This initial study would only include one switch per text and neutral texts would be placed between valenced texts in order to "reset" the participant's mood state. (If hypotheses are borne out, future studies could increase the number of switches to determine how many words are necessary to build up a given mood state during the reading aloud task.)

the design here is not entirely clear to me. what, exactly is the dependent measure? And, what is the prior work to suggest that only a few trials are needed to get a reliable measure? It sounds like the idea is to have participants read relatively long passages with a switch in between. However, how many "switches" would we have to analyze in this case?

-can you please paste below an example of the stimuli which includes the "switch"?

additionally, can you please provide much more detail on the design of the task? In terms of length of text passages, number of passages, when mood will be assessed, how data will be analyzed (in terms of coding the speech and exactly how errors/ response times will be defined)? Related to this, we will be going over how we will be analyzing the rwe-alpha task soon, so, feel free to wait until after that discussion to follow up on this part, if you prefer.


> 
> This research project was originally conceived as a spin-off of rwe-alpha with a separate text set. However, the counterbalancing on rwe-alpha is quite complex and adding a second set of texts plus EEG to the existing protocol seems ill-advised. Therefore, this project may need to be it's own "Part 1B," but could easily be connected to Part 1A for the three studies launched in June 2021.

sorry, not totally clear. Is the idea to have a study that is the same as rwe-alpha for part 1a and then a different part 1b? is there a strong need for the measures in part 1a for this new proposed study? To me, it seems like the newly proposed study could be run as a single (not two-part) study, with limited questionnaires assessed, and without the multiple EF tasks assessed. 

> 
> In addition, I would like to add a measure of current mood state in order to assess whether the impact of the textual mood states (and switches) is impacted by the participant's mood at the time of reading. One existing open-source option is the [Brief Mood Introspective Scale (BMIS)](https://mypages.unh.edu/jdmayer/brief-mood-introspection-scale-bmis). However, as the intent is to actually _modify_ participant mood states during reading and then surprise them with a switch, it would be useful to gather the participant's mood multiple times during the task and the BMIS is too cumbersome for repeated use. A simpler instrument, such as the self-designed and color-coded "mood check" (see instruments/mood-check.png) could potentially be used in place of or in combination with the BMIS.

agree that using BMIS (or something similar) could be useful. Also, that it would be interesting to assess more dynamic changes in mood. However, we need to think about when/how mood would be assessed during the reading task. This might be rather difficult to do in practice. 

> 
> **Funding:**
> Require guidance on short-term funding. Some longer-term possibilities include:
> 
> * NIH R03
> * NIH R21 (for greater RWE)
> * NSF CogNeuro
> * NSF PAC (Perception, Action & Cognition)
> 
> **Authors:**
> George/Jess, Angela?, Sarah?, Olivia?

Looks good as initial suggestions. You should be first, and me last, though. 

Meta-comment: what should the recommendations be in terms of suggesting other authors. I would think that it would in general. be recommended to reach out to those you think might be a good fit before posting, right? And then, you also @ them when posting. With that said, I don't think we want to make it a requirement to reach out to those you suggest beforehand, but they should at least be @ed, right? (I know you can't do that for Olivia and Angela yet).
-Want to also make clear what the proposed author order is in the author list as well.

> 
> **Milestones:**

I think what is missing in this section is things like "piloting of experiment" and "XXXX aspect of data analysis" and 
> 
> _Conferences_
> 
> * Poster of initial behavioral findings (hypotheses 1+2) submitted to Social & Affective Neuroscience Society (poster deadline mid-December 2021)
-this would be a pretty tight deadline, in my opinon. Possible, but I would make this a "goal" not an "expectation". There is going to be a fair bit of trial/error piloting to get the task working right at first, and only then can you collect data, and you will need a fair bit, and then analyses will be a learning process too. I would probably shoot for a Spring submission. But, open for making a dec submission a "stretch goal"!!

> * Poster of initial conflict monitoring findings (hypothesis 3) submitted to Cognitive Science Society (poster deadline mid-June 2022)
>   (Note: poster deadline for Cognitive Neuroscience Society is late-November 2021; however, eligibility of non-students is unclear. The poster slated for submission to CSS could be submitted first to CNS if sufficient data were available by the submission deadline.)
> * Abstract of findings for inhibitory control and mood congruence (hypothesis 4) submitted to Society for Affective Science (abstract deadline likely late-November 2022)
> 
> _Publications_
> 
> * Paper submissions of behavioral hypotheses (1+2), conflict monitoring/instantiation of control (3), and inhibitory control/mood congruence (4) as adequate data are collected and analyzed.  Some potential publishing outlets:
>   -Motivation and Emotion
>   -Affective Science
>   -Cognitive Science
>   -Trends in Cognitive Sciences
this is more for reviews, etc.
meta-comment: at some point (low priority) we should make a list of journals that we will typically submit to, and which ones are good for which papers.
>   -Cognitive, Affective, & Behavioral Neuroscience
great journal for something like this, if neural data included.
>   -Journal of Cognitive Neuroscience
>   -Emotion
>   -Glossa Psycholinguistics (new journal)
Always be weary of new journals... ;)

meta-comment: I actually don't think we need to worry so much about the specific journal to submit to for the brainbox. Fine if people mention it, but I think the thing we want to focus on is the content/scope of the manuscript(s)
-I know we have been talking about how the project we initially launched have several papers planned, but keep in mind that those are central, core project lines in the lab, so they need to have that. I think that the general assumption for a new brainbox submission is that it will typically lead to one (and MAYBE two), but I think the general focus should be on one paper. My thoughts here could change, but for now, I think that is a good way to go, so that students can maintain focus on a given paper.

**Comment by georgebuzzell** [comment]
@jessb0t specific comments above. But overall, I think this is a great project idea. Big picture, I think that the neural measures should be dropped from the initial study, in order to focus in on identifying some really solid behavioral effects. Additionally, what I think is most needed right now is more detail on the design, and in particular, exactly how behavior will be analyzed. Lastly, I am concerned about the number of trials for looking at switches. However, it seems like maybe that is a secondary aim in the initial study? If so, I think that is fine, but please refine.

Look forward to seeing the next version!

**Comment by jessb0t** [comment]
@georgebuzzell: Thank you very much for this helpful and constructive critique!  I have revised the proposal above (replaced initial entry of this GH issue) in light of your comments.  Major changes:
- I have dropped the neural measures entirely as I fully agree with your notes.
- Given your comments about not trying to connect to rwe-alpha, I have envisaged a standalone, fully asynchronous study.
- I have fleshed out the main question, predictions, design, and milestones, with a focus on a few key areas of interest (less is more! 🙂 ).
- I have added a possible method for dynamically analyzing mood state (that is, I have modified the design to make it possible to analyze this down the road if additional research finds that my idea holds any water...further reading is required and I think this should be lower priority than looking at the main questions proposed here).
- In light of your comments about authorship and the absence of conversations with potential collaborators to date, I have left this TBD.
- I appreciate the guidance on timeline and would prefer to aim for the spring (rather than push for a submission in December).

Regarding the number of switches, I have shortened the passages significantly, but maintain that we want just one switch per passage.  My reasoning for this is that we do not have any data on how long it takes to build up a "mood state" during reading.  Prior research I found that experimentally induced mood either used the [Velten method](https://dictionary.apa.org/velten-technique) or induced mood by music or video.  I am concerned that if we include multiple switches in each passage, we will effectively be creating a "muddy" semantic priming task, making it difficult to isolate any behavioral effects of shifts in semantic valence.  I would welcome your advice on how to conduct a power analysis so that we can determine if my proposed design is feasible (that is, how many texts and how many participants are needed to get a reliable measure for analyzing the second hypothesis?).

While most of your questions are answered by my update above, there were two that required some additional detail:

> is there no work to show something similar at the passage level? If no, then this is "low hanging fruit" and an obvious thing to test. But, please be sure it has not been done?

I have found only two articles in this vein, which I have added to my reading list:
1. [How valence affects language processing: Negativity bias and mood congruence in narrative comprehension](https://pubmed.ncbi.nlm.nih.gov/19487747/) - Focus is on valence of story ending, not internal story content.
2. [Effects of Valence and Emotional Intensity on the Comprehension and Memorization of Texts](https://pubmed.ncbi.nlm.nih.gov/30774618/) - Focus is on comprehension and memorization, not performance.

> any work to show how this differs based on individual differences? I think there might be work on how this differs for anxious/non-anxious. I would be interested in effects based on both trait and state anxiety.

There seems to be a decent amount of literature devoted to lexical decision tasks and correlations with state or trait anxiety.  I have added the following to my reading list, but there are plenty more to choose from:
1. [Biased cognitive operations in anxiety: Accessibility of information or assignment of processing priorities?](https://doi.org/10.1016/0005-7967(91)90009-R)
2. [Effects of Emotional State on Lexical Decision Performance](https://doi.org/10.1006/brcg.2000.1248)
3. [The influence of anxiety on lexical and affective decision time for emotional words](https://doi.org/10.1016/0191-8869(89)90077-9)
4. [Examining attentional biases underlying trait anxiety in younger and older adults](https://doi.org/10.1080/02699931.2013.802224)

I look forward to your feedback on these revisions! 🤓

**Comment by georgebuzzell** [comment]
@jessb0t Looking at this now. But meta-comment (and sorry if I am misunderstanding brainBox currently). I see that you have a branch for the proposed study. But, I think that is new, and I think the original readme for BrainBox said to just create an issue. I actually might suggest that we change the workflow so that if you have a new idea, you create a branch, and then write your idea out in a readme for that branch. then, you do a PR to allow easier review/commenting. It is just sort of difficult to provide a detailed review via comments on issues, as I have to quote each piece. Also, when you make updates, it is hard to tell when/where if you edit an existing comment. I think this would all be easier to track/review if the standard workflow was to create a branch, write the idea in a readme, and do a pr, then, we have comments on the pr. Thoughts on all this? Can we maybe give it a shot for any additional rounds of feedback here? That said, I will not go ahead and give feedback on the issue as it is! :)

**Comment by georgebuzzell** [comment]
@jessb0t PS to the meta-comment: The readme and pr route would not only be easier to review on my end but, of equal importance, I think it makes tracking the provenance of an idea, what was said/suggested when and by who, much more straightforward. this is crucial to ensure you and other lab members (or external contributors) all get the credit they deserve for the ideas they create. :)

**Comment by georgebuzzell** [comment]
@jessb0t to facilitate review, I am pasting your entire issue from the top here, so that I can insert comments throughout. :) (if in a PR I could just add them directly) ;)

comments from GB in **bold** 

Question:
Broadly: how does our perception of emotion impact our behavior? Specifically: in what ways does semantic valence (and known priming effects) impact performance when reading a text aloud, and how is such performance moderated by the relationship between the participant's own mood and the mood that the passage endeavors to evoke?

**Minor comment: Curious if "perception of emotion" is actually what you mean here, as opposed to something like "experience of emotion"? To me, "perception of emotion" implies attention being directed to the emotion itself (and conscious awareness of the emotion). I am not saying that is not something worth study, just that from the prior discussions, I am not quite sure that is what you are actually focusing on?**

Summary
According to Hutchinson and Barrett, 2019:

Valence and arousal might be better thought of as properties of consciousness, rather than properties of emotional episodes per se...

**Not totally clear what is meant here by "properties of consciousness". This is probably a minor comment though...**

In this way, a person's mood might be viewed as a filter through which they experience the world. When faced with conflicting emotional stimuli, an individual might be required to deploy inhibitory control in order to ignore their own mood and interact with the stimulus. One would be expected to perform better (faster, more accurately) on any task when they do not have to expend resources on such inhibitory control. Likewise, switching between ends of the valence spectrum may operate similarly to switching between tasks, with emotional switches serving as stumbling blocks to the smooth execution of a task.

**LOVE this prior paragraph. THIS is speaking my language! :)**

This research idea dives into these questions by exploring speed and articulation accuracy in an ecologically valid context: reading a text aloud. Texts are designed to be heavy on emotional words, allowing performance to be analyzed according to elements of the stimuli themselves (passage valence and direction of the emotional switch) as well as on individual differences (mood state, emotion regulation abilities, and measures of anxiety).

**Great!**

Lit Review:
(See literature/lit-review.xlsx.)

Emotion has moderator effects on cognition and behavior, with processing advantages often seen for positive, visually-presented words:

slower processing of negative stimuli (versus positive) as seen by slower response times on negative target words in lexical decision tasks (Kazanas and Altarriba (2015))
positive primes in lexical decision tasks ostensibly cause spreading activation in the semantic network, thereby shortening response time for related target words but slowing response for unrelated target words; negative primes seem to cause an emotional reaction inhibition that precludes any change in response time for related/unrelated targets but enables faster processing for unrelated targets than a positive prime, with its larger network activation (Sass et al. (2012))

**Interesting... :) Of course, I do still wonder how this might differ as a function of individual differences in trait affect. E.g. those prone to negative affect, do they show more pronounced effects, but in the same direction (what I would predict based on what you have written above) or do they show a reversal of the effects. ***

smaller N400 amplitudes for pleasant v. unpleasant adjectives in silent reading may indicate that pleasant words are semantically-integrated more easily, possibly as a result of congruency with default, mildly positive mood states (Herbert et al. (2008))

**Without more details/context, not sure I totally understand the above statement. I am familiar with the n400, just not the details of the study above. Not sure this is crucial though.**

Hypotheses:
Prediction 1A
Positivity bias that is ostensibly caused by the larger network activation of positive words would impact reading speed and error rate in valenced passages, with faster reading speeds and fewer errors in positive passages than negative passages.

**OK, cool. So hypothesis one is basically an extension of the prior work with artificial word presentation to more naturalistic passage reading, right? If so, great! And also, if I understand correctly, then I think we would also want participants to probably perform one of the classic tasks as well. That is, have participants do one of the classic/artificial tasks on which you are basing your hypotheses, but then, also have them do the more naturalistic passage reading. This is a "safer" study design, as you then should be fairly guaranteed to get a significant effect (replication) of the prior work, even if the more novel passage task does not work. It also helps to rule out some possibilities if we find nothing with the naturalistic task. 

To make this easier to do, there is no need to reinvent the wheel and also create the classic task from scratch. Instead, see if one is available online. Or, we can reach out to the authors to ask for a copy (not yet though). This will only work though if the task is implemented in a format we can easily use. I believe we have access to ePRIME, and MAYBE presentation. We also have matlab/pscyhtoolbox, and of course, psychopy.** 

Prediction 1B
Participant mood state may moderate the relationship between passage valence and reading aloud performance, with faster reading speeds and fewer errors in passages that more closely match the participant mood state (that is, participants in more positive mood perform better on positive passages than participants in more negative mood, and vice versa).

**Valid hypothesis. You might also identify a more nuanced pattern, based on some of the things you wrote above. In particular, you might instead find that a postive mood leads to faster reading of positive passages, and, that a positive mood actually buffers against the "main effect" of negative passage valence, such that those with a positive mood read the negative passage slower than their reading of the positive passage, but still faster than those with a negative mood reading the negative passage (based on the theory of spreading activation you mentioned above). OK, to not fully flesh out this alternative hypothesis now, but please do give it at least some thought and include it as an alternative outcome. Note: the BEST STUDY you can possibly run is one where the results are interesting and of value regardless of how they come out. This occurs when you set up a study that is not just one-sided (i.e. I have hypothesis A and that will be confirmed if I see pattern x). But instead, you set up your study such that you are able to rule in favor of two competing hypotheses (e.g., There are two competing hypotheses, A and B, and prior work is not sufficient to determine whether A or B is a better explanation of nature, thus, I set up a study that will yield one of two patterns of data, X or Y, If we see pattern X, then this would support hypothesis A, but if we see pattern Y, this would support hypothesis B). Taking this tact will force you to think critically about the different ways that nature might be working, as opposed to latching onto a "pet theory" and creating straw man tests to confirm it. That said, this is not an easy process!**

**Also, I think it is great to focus primarily on state mood. However, Please also plan to include measures of trait and similar tests. Relatedly, we migth also propose a model where you test the effects of both trait and state mood on reading and interactions with passage valence (all in the same statistical model). OK to not flesh out the statistical details for now, but do flesh out the measures/procedures.**

Independent Variables
- variable 1: valence rating of passage-half, calculated using the "emotional tone" rating from [LIWC](https://liwc.wpengine.com/) {continuous}
- variable 2: whether passage-half was pre- or post-switch {categorical, treated as binary}
- variables 3+ (1B only): participant measures on initial questionnaires (BMIS, etc.) {continuous}

**above looks good (for now)**

Dependent Variables
- speed of reading aloud: syllables (required to read the text, not necessarily as produced) / second {continuous}

**above is not totally clear. is this the average speed of reading syllables averaged across the passage half? I think that is interesting, and likely something we should explore, but at least in addition to this, a good measure would probably be total time to read a passage half (after matching physical length), maybe?**

- percentage of disfluent syllables: number of disfluent syllables (repeated syllable, pre-syllable pause or interjection, and/or mispronounced syllable) / syllables required to read the text aloud {I think this can be either binary or continuous, depending upon how we run the analysis?}

**perfect! Like this :) And, I think we would probably code each syllable as correct/incorrect (binary) but, the actual dv would be th total number of dysfluencies, so continuous, right? Or am I missing a piece here?**

Prediction 2
Larger network activation for positively valenced words and the compensatory mechanisms that seem to be triggered by negatively valenced words would operate to benefit negative>positive switches more than positive>negative, with a greater likelihood of disfluency at a positive>negative switch than a negative>positive switch.

**it might just be getting late for me... :) but I dont 100% understand the above? That said, given that everything else here is really solid, consider this a minor comment (but one to try to clarify nonetheless, please)**

Independent Variable
type of switch (pos>neg or neg>pos) {categorical}

**great!**

Dependent Variable
percentage of disfluent syllables across three factors:
    a) within the switch word group (two words prior to the switch, the switch itself, and two words following the switch)

**How do we delineate the switch per se? It is intuitive how a sentence as a whole can have a clear valence, and also how individual words can have a clear valence, but, what if the last few words of the last sentence in the negative valence sentence are not really negative per se? Are they still considered pre switch? Or, is that the "switch itself" you are referring to? If that is, then would this not be an issue that you are relying on prior norms of the valence of passages, which likely rate the whole passage ad negative? Ultimately, I guess what is confusing to me is how the "switch itself" period is defined? It would seem that the most straightforward approach is to just list everything up until the last word in the passage normed as "negative" (or positive) as the pre switch period, then the first word and beyond for the following passage is post switch, and, you would not really have a period of words that are called the "switch period". But maybe I misunderstand?**

    b) in the pre-switch word group (the five words that precede the switch word group)

****same question, not clear what the switch word group actually is** That said, I do like the idea of analyzeing the data around the switch at increasingly large windows. Totally agree this is the way to go. Just not clear about this "switch word group" part**

    c) in the post-switch word group (the five words that follow the switch word group)

**ditto**

**Additionally, i might suggest expaning even further to the whole sentence before/after, and maybe, a fourth level for the two sentences before/after**

Note 1: I feel that syllables form a logical 'unit' in which to measure disfluency.  However, as semantic integration is required to prepare articulation during reading aloud, I propose that we slice on the word level for analyzing disfluency around the switch.

**I would suggest assessing dysfluencies at both levels. but fine to start with word level**

Note 2: As a starting point, I selected groups of five words; this may need to be revised.  My future reading list includes [this article](https://www.frontiersin.org/articles/10.3389/fpsyg.2015.01432/full), which may provide better guidance.

**see above comments** 

**also, I think that this will ultimately be born out in pilot data, and, you have thought this out fairly well so far, and worth seeing what some pilot data shows!**

Design:
Prior to the experimental task, participants complete several questionnaires:

Demographic questionnaire
Emotion Regulation Questionnaire (ERQ)
Interpersonal Emotion Regulation Questionnaire (IERQ)
Beck Depression Inventory (BDI-II)
State-Trait Anxiety Inventory (STAI)
Brief Mood Introspection Scale (BMIS)

**these are good, I think you also want something that gets a bit more at trait affect directly. I think there is a questionnaire that is specially designed to assess trait negative affect. The ARI might also be good**

After completion of the questionnaires, participants are prompted to tell a brief story (~1 minute) about something that has already happened to them that day; this brief narrative is recorded. (Further research is needed to determine how best to ensure that this narrative prompt is neutral.) For the initial analyses, this brief narrative does not serve any specific purpose; however, future analyses may be able to correlate qualities of the narrative with the BMIS measurement.

**Can you pleaser give more details about the purpose of this? If it is really just an "extra" thing you want to do, then it should really go at the end of the other study procedures, so as to not interfere. Also, what exactly are you trying to asess here, as that is crucial for the instructions. I.e. do you want to know how they felt about things? Are you minging their speech for valence of words? what?**

**This seems like an extra thing, so probably not needed to spend much time developingt this before opening the repo for the project. But, i would do this at the end (if done)**

The main study task involves the reading aloud of texts (~200 words each) that are designed to evoke a certain mood state in the reader (positive/negative) at the start of the passage, but include a single valence "switch point" where the mood of the text suddenly reverses. (Examples: pos>neg passage and neg>pos passage.) Blocks of texts, perhaps three, are presented back-to-back in such a way that "switch points" are passage-internal (not between passages). Between each block, a "break" is provided during which the participant is asked to reflect upon a thought/memory evoked by their reading, similar to the initially recorded narrative. The primary function of these breaks is to recalibrate the participant's mood back to a default state before the next block. A secondary function, if future analyses are able to correlate narrative qualities with the initial BMIS measurement, is to offer the possibility of identifying more dynamic changes in mood during the reading aloud task.

I think you want to be careful here. There might be a bias towards the first/second pasage in terms of what they recall. Or, there might be an overall bias to recall something positive vs negative. Thus, if the goal here is to really "recalibrate" before the next block, I dont think this is the way to go. If that is the goal, then you should have them read a very short neutral passage before the next block.

**But it seems like there is a second goal here, wehre you actually want to see what they recall. but again, there will always be two valences present, so are you really assessing their accuracy in attending to mood, or just their bias to remember postive/negative things if you ask them to only recall two things. If you want to assess bias, then yes, ask them to just recall one. But if accuracy of introspection to mood is the goal, then you need to explicitly ask them to recall the something from each passage. But again, you need to be careful here, as then you are setting up a design where you make the participant very aware of the study design (the mood switc), and that might yeild qualitatively different results than if you left everything implicit. Since you are building on prior lab tasks, what do they typically do? do they have manipulations/instructions that bring awareness to the affect of the words? I would assume no and that it is typically implicit? And, if I am correct here (i could be wrong though) I would suggest dropping these explicit questions between blocks. Instead, you can have another passage or two at the end of the experiment where you explicitly ask them about mood etc to get that measure of attention to mood/introspection without confounding the primary study. does that make sense?**

**it seems like you propose only three blocks (three switches), yes? I really think that is too few if you want to have a reliable measure of switching. Typically, we want something much higher for behavioral data, like at least 20 or more events, but I understand that is not feasible here. But perhaps you can get it up closer to 10? how long would that take in that case? If too long, maybe at least 6 or 8?**

After the reading aloud task, the participant is prompted to share their overall impression of the activity (~1 minute), which is recorded. They then re-complete the BMIS.

I believe that, with careful setup, this study could be completely asynchronous as PsychoJS now supports microphone input.

**all for it if possible. Piloting will make that clear. no need to worry about this too much for now. lets keep it an open question if async/synch. Regardless, intial piloting should be synch.**

Funding:
Require guidance on short-term funding. Some longer-term possibilities include:

NIH R03
NIH R21 (for greater RWE)
NSF CogNeuro
NSF PAC (Perception, Action & Cognition)
Authors:
George/Jess, co-authors TBD

**List your name first though!**  :)

Milestones:

Piloting
In order to get to a piloting stage, we will need to: perform power analyses to determine an appropriate number of participants to analyze the two sets of predictions, submit the IRB protocol (including recruitment materials, sample stimuli, and informed consent) and receive approval, create the full stimuli set, determine the neutral prompts for the "breaks", code and test the Pavlovia experiment, and setup the REDCap project. It is estimated that this could be completed in the second half of the fall semester.

**unfortunately, power analyses are really difficult if you are doing something really novel with no idea of the reliabilty of the measures or effects sizes. So, I dont think we need a formal power analysis here to get started, we can just use ball park numbers to get started. You are going to need an irb early, to allow piloting, and then, you will 100% have to revise the irb when closer to running the real study. So, a high priority is to get a basic irb approved**

Data Coders
All recordings of the valenced passages must be coded for errors. I believe that this should be done independently by two coders who are blind to the experimental question, to decrease the risk that they will be biased in their evaluation of what constitutes a disfluent syllable. An example tool for coding is provided here. Ideally, at least one coder would have completed an introductory Linguistics course. Training of coders should take place in the second half of the fall semester so that they can begin coding recordings as soon as pilot data begin coming in.

Analysis 1A
This analysis involves two multiple regression analyses (one for reading speed, one for accuracy), plus individual t-tests on the assumption the F is significant.

Analysis 1B
This analysis involves a slightly more complex model than 1A. My explicit prediction only involves the BMIS measurement, but I believe that we will want to explore how scores on emotion regulation and state/trait anxiety measures might impact the model.

Poster: Cognitive Science Society
This poster would share the results of the 1A and 1B analyses. The submission deadline is anticipated to be mid-June 2022.

Analysis 2
Because each passage only contains one switch, a sufficient number of participants will need to be enrolled before analysis can begin on the switches themselves. Further work is needed to complete a power analysis in order to determine the appropriate number of participants. We may also need to refine the scope of each "switch word group" (see note above). I do not have an explicit prediction on how mood state, emotion regulation abilities, and state/trait anxiety measures might impact performance, but we may also want to run exploratory analyses that include these measures.

Publication
This paper would share the results of analyses 1A, 1B, and 2. The goal is to have it submitted by early fall 2022.

**SOunds like a good goal to me!**

Future Analyses
Pending further research, it might be possible to use the narrative "breaks" as a dynamic measure of mood state. The pre-task narrative could potentially be correlated with the pre-task BMIS measurement by looking at acoustic data and lexical choice. If a reasonable correlation were found, we could then analyze the content of the narrative breaks to better understand how a participant's mood evolved over the course of the task. The following are on my reading list to better understand feasibility, but I have read that similar endeavors have proven promising...

**Please see the comments above about the issues introduced by asking questions during the breaks. Just keep in mind that you want to keep your current, well planned hypotheses/analyses the best shot and to not compromise them with stuff that "might" be interstin for the future. YES, you totally want to include things in your study that would lead to future work, but, you want to do so in a way that does not interfere with the main analyses. See comments avove**

The Psychological Meaning of Words: LIWC and Computerized Text Analysis Methods
The role of voice quality in communicating emotion, mood and attitude
Human voice perception
A dimensional approach to vocal expression of emotion

**Comment by georgebuzzell** [comment]
OK, @jessb0t . First thing I have to say that this is really impressive. The origonal idea you came up with was great, but it is really impressive how much further you have developed and refined the idea(s) here. Really, very impressive.

It seems like this is just about ready to "go live" with opening a repo for this project. Almost all my comments above are quite minor. That said, probably best to address those officially with one more revision here, which I will almost certainly approve right away and you can open the project repo for this. 

Again, really impressive work here. But, at this point, I am not surprised! ;)

**Comment by jessb0t** [comment]
@georgebuzzell
Will address meta comments next week.  Good points and I have some additional thoughts! 🤓

**Comment by jessb0t** [comment]
@georgebuzzell

A more thorough response to your feedback to come, but I wanted to briefly respond to some of the larger questions!

> above is not totally clear. is this the average speed of reading syllables averaged across the passage half? I think that is interesting, and likely something we should explore, but at least in addition to this, a good measure would probably be total time to read a passage half (after matching physical length), maybe?

Wouldn't these be equivalent (if length of passage-half were measured in syllables, which I would argue is a more basic unit of articulatory measurement, rather than letters or words)?

> And, I think we would probably code each syllable as correct/incorrect (binary) but, the actual dv would be th total number of dysfluencies, so continuous, right? Or am I missing a piece here?

Correct.  So, for instance, if a participant both hesitated before a syllable **and** mispronounced it, that would be an "incorrect" syllable.  Assuming the passage-half were x syllables long and this were the only performance error, then we would be at 1/x disfluency (a percentage).  Can you use a percentage as a continuous measure?  Otherwise, we could just do a count of disfluencies like you suggest, but we will need to correct for differing lengths of passage-halves.

> How do we delineate the switch per se? It is intuitive how a sentence as a whole can have a clear valence, and also how individual words can have a clear valence, but, what if the last few words of the last sentence in the negative valence sentence are not really negative per se? Are they still considered pre switch? Or, is that the "switch itself" you are referring to? 

In looking at the switch point, I was imagining we would expand our focus beyond **just** the switch word as it seems likely to me that, in the context of reading a passage aloud, we would get performance shifts in the vicinity but not necessarily on the word.  Some prior research has looked into how far ahead people begin semantic and phonological encoding (so they understand an upcoming word and prepare to articulate it).  For instance, if a participant is three words away from the switch word in terms of actual reading aloud production, but they have already **seen** the switch word, and assuming this does mess with their performance, they might stumble three words before the actual switch word.  Alternately, they might recruit all the necessary resources to do that correctly...and then stumble after they successfully make it past the switch word.  My intent is to look more holistically at performance in a "regular" text (which has a given tone and whose tone varies at a particular point in time) rather than creating priming duos where a positive and a negative word are back-to-back.  So the buckets of five words were a starting point, but we can certainly narrow/widen those groups during analysis.

> Can you pleaser give more details about the purpose of this? If it is really just an "extra" thing you want to do, then it should really go at the end of the other study procedures, so as to not interfere. Also, what exactly are you trying to asess here, as that is crucial for the instructions. I.e. do you want to know how they felt about things? Are you minging their speech for valence of words? what?

The original thought was that our blocks of text need breaks between them, otherwise, it will quickly become self-evident that we're flopping back and forth between positive and negative paragraphs.  At first, I thought we could just create "neutral" texts.  But then, imagining myself in the shoes of the participant, I thought it would be nice to have a "real" break where you didn't have to read something aloud.  This led me to imagining that, in the real world, people just talk between tasks all the time.  As long as our prompts are neutral, the participant can say whatever they like during the break...we just want them to talk.  For the current study, the purpose is separating blocks of reading.  Down the road, though, we might be able to mine these breaks for more interesting information.  So, two birds, one stone: less stimulus creation now (because we wouldn't need to create any stimulus for the "breaks") and the possibility to actual analyze dynamic changes in mood down the road. 🙂 

> I think you want to be careful here. There might be a bias towards the first/second pasage in terms of what they recall. Or, there might be an overall bias to recall something positive vs negative. Thus, if the goal here is to really "recalibrate" before the next block, I dont think this is the way to go. If that is the goal, then you should have them read a very short neutral passage before the next block.

Perhaps we can come up with something more neutral?  I went with "reflection upon something evoked by the passage" simply to maintain the ecological validity: people do work, then talk to one another...and what they are working on typically serves as primary fodder for their conversation.  I had no intention of mining the content of these breaks for data on what they did or did not recall from the passages and certainly would not want to create unintentional biases.

> it seems like you propose only three blocks (three switches), yes? I really think that is too few if you want to have a reliable measure of switching. Typically, we want something much higher for behavioral data, like at least 20 or more events, but I understand that is not feasible here. But perhaps you can get it up closer to 10? how long would that take in that case? If too long, maybe at least 6 or 8?

Each block would include three passages, so we each block would be:

pos>neg | neg>pos | pos>neg
or
neg>pos | pos>neg | neg>pos

in an effort to make the flip-flop nature of the texts less apparent to participants.  You mention elsewhere that a power analysis would not be appropriate, so I think we should aim for something around your proposed 20.  If we did 6 sets of 3, we would have 18 switch points and participants would read approximately 3600 words in total for the task.  Or we could do 7 sets of 3, for 21 switch points, and approximately 4200 words of reading?

> So hypothesis one is basically an extension of the prior work with artificial word presentation to more naturalistic passage reading, right? If so, great! And also, if I understand correctly, then I think we would also want participants to probably perform one of the classic tasks as well. That is, have participants do one of the classic/artificial tasks on which you are basing your hypotheses, but then, also have them do the more naturalistic passage reading. This is a "safer" study design, as you then should be fairly guaranteed to get a significant effect (replication) of the prior work, even if the more novel passage task does not work. It also helps to rule out some possibilities if we find nothing with the naturalistic task.

Yes, I suppose hypothesis 1 is effectively seeing if the same pattern holds in a more naturalistic environment.  You are proposing that we add a full lexical decision task to the study?  Kazanas and Altarriba used 48 words pairs (prime/target) blocked by valence and all words were valenced (that is, they had no neutral stimuli).  Sass et al. ran their experiment in German.  Yap and Seow used a more balanced set of stimuli, but theirs was not a priming task (and did not find any significant differences between positive and negative stimuli).  I see your point about including an actual replication, but I fear that it will draw heavy attention to the fact that there are valenced words in the reading passages?  And possibly tire our participants out too much?  If you don't think these are major risks, we can look to see what other LDT studies exist using both valenced and neutral stimuli, what they found, and whether it would be feasible to 'borrow' the task for a replication.

**Comment by jessb0t** [comment]
@georgebuzzell As discussed, I have moved the pitch into the readme file of the rwe-valence-jess folder here on brainBox.  It has been revised in accordance with your latest feedback, but please also see my notes in the comment above.  Thank you! 🤓 

(P.S. I uploaded the readme in multiple commits so that we have the full history on that file.  Replacing original comment in this issue with a note to that effect.)

**Comment by jessb0t** [comment]
@georgebuzzell 
I renamed my branch, folder, and issue to "readAloud-valence-jess".  I didn't realize until after the fact that this effectively deleted the old branch, so I had to rebuild it.  Although the commits aren't superb, I re-created the chronology on the readme file so we can still see its evolution in the commits.

Please see the two comments above since your last review.  I have also initiated a draft pull request to see how the workflow feels for you!

Lastly, do you know of any good measures for reading skill in adults?  I'm thinking we may want to assess this, even on a simple scale, so that we can ultimately exclude the results (at analysis) for anyone whose reading level is too low (however we decide to define that).  I did some preliminary research, but didn't find any obvious instrument that is used in typical, English-speaking adults.  Would value advice on where to look!  Thanks!

**Comment by georgebuzzell** [comment]
@jessb0t please see the long (sorry) comment I added in my review. TLDR: there are a number of details we will still need to figure out, but I officially approve moving forward and creating repos, assuming you agree with the addition of the traditional task of some form (tbd outside of brainbox). If we are not yet in agreement there (which is fine) we should do another round of edits in brainbox; otherwise, please charge ahead and we can continue design discussion in the official repo! Really, really great (and impressive) work!

**Comment by jessb0t** [comment]
Closing this issue as discussion has moved to #5.
